Jenkinsfile

```diff
@@ -23,6 +23,7 @@ ansiColor('xterm') {
             sh """sudo -E ci/pipeline jenkins"""
         }
       } finally {
+        currentBuild.description = "<h1>Archives</h1><ul><li><a href='${env.BUILD_URL}/Scapegoat_Report'>Scpegoat Report</a></li></ul>"
```

Comment:
why not just `archive` it in the jenkins file?

Reply:
I'd like to upload to S3 and post a better summary.

Reply:
> I'd like to upload to S3 and post a better summary.

This feature is useful if you attach HTML reports (with graphics and whatnot) to the build page. Otherwise, Jenkins archive is sufficient.

Reply:
There is a nice plugin that would do this for us. Unfortunately there was no reaction to this so far.